app/scripts/controllers/table.js

```diff
@@ -244,38 +244,19 @@ app.controller('TableCtrl', ['$location', '$scope', '$modal', 'ForamAPIService',
     // filter forams - get total number and then load forams
     var filterForams = function () {
       $scope.loader = true;
-      ForamAPIService.getForamsInfo(flatFilters)
+      ForamAPIService.getForams(flatFilters)
         .then(function (res) {
           if (res.status < 400) {
```

Comment:
Jest to coś co Eryk bardzo konsekwentnie stosuje... Też mi się wydawało dziwne, ale wolałbym nie ruszać - jak już, to naraz zewsząd usunąć